code/datums/tracking_sound.dm

```diff
@@ -0,0 +1,114 @@
+GLOBAL_LIST_EMPTY_TYPED(sound_spatial_trackers, /datum/sound_spatial_tracker)
```

Comment:
we should make this a 1024 entry list, and just access it by channel
then just null the entry instead of -=